src/workflow/snakemake/rules/rule_node.smk

```diff
@@ -0,0 +1,84 @@
+import os
+
+from src.workflow.snakemake import scripts
+from src.workflow.snakemake.format import formatter
+
+
+def create_node_rule(node, benchmark=None):
```

Comment:
(future) perhaps this is the ideal place to extend with "extra" rule elements, e.g. envmodule, singularity, conda, log, benchmark etc

Reply:
Added a comment in the file.